### DISCUSSION TITLE
403 error

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#388] [Q&A (Community Support)]
Hi today on 40% of pages I have in Panopticon have error
`An error occurred updating the site information.
Client error: `GET https://*******/api/index.php/v1/panopticon/core/update?force=1` resulted in a `403 Forbidden` response: <html> <head><title>403 Forbidden</title></head> <body> <center><h1>403 Forbidden</h1></center> <hr><center>openresty</c (truncated...)`

Manually refresh is not changes notification. 
![Akeeba-Panopticon](https://github.com/akeeba/panopticon/assets/17553018/26d976ea-f9f7-4dc1-9630-c3810cd4a42e)

**Comment by nikosdion** [comment]
This is a problem with your server. Ask your host why the request is blocked, not the Panopticon community. We don't run your sites' servers.
